src/attachments/modifyImage.spec.js

```diff
@@ -15,6 +15,7 @@ describe("image modification", () => {
   const thumbFile = `${__dirname}/__tests__/duck_thumb.png`;
   const tinyFile = `${__dirname}/__tests__/duck_tiny.png`;
   const scaleResizeFile = `${__dirname}/__tests__/duck_500x400.png`;
+  const wrongImageFile = `${__dirname}/__tests__/wrong.png`;
```

Comment:
I'd call this `missingImageFile` or use `wrong-image.png` or `wrong-image.jpg`, if you want the modification process to break with an image file that is no image file